### DISCUSSION TITLE
Help on using media icons in rc.5

### DISCUSSION THREAD

**Discussion by jevantang** [vuejs/vitepress#2860] [Q&A] [answered]
I don't know anything about programming, so I'm not sure how to implement the media icons in rc.5?

https://vitepress.dev/reference/default-theme-team-page
This page is also down, and it doesn't look like there's a new version of the guide yet.

I am using twitter and youtube icons!

thanks

**Comment by brc-dd** [comment, marked as answer]
Try with rc.6 It should work fine now.
